### ISSUE TITLE
RuntimeError: maximum recursion depth exceeded using python3.4

### ISSUE THREAD

**Issue by gusantor** [SelfDestroyer/pyMOD-OLED#5]
Hello, I'm trying to use this library with my A20 and a generic OLed 128x64 screen, testing with Hello_World.py

I can use it and it performs ok with python2.7

I have a virtual environment using python3.4 , I have to run script as sudo, to be able to access /dev/i2c-0

when I run, as sudo, with the virtual-env python3.4 I receive error as shown below 

`RuntimeError: maximum recursion depth exceeded`

If I just print a dot or line, not words, and comment out lines regarding Font, it works on both python2.7 and python3.4

It's just happening to me ?

please can you give some hint on how to solve ?

thanks in advance

```
Traceback (most recent call last):
  File "asistente-google/hotword.py", line 64, in <module>
    f = Font(2)
  File "/home/mauricio/virtual-envs/asistente/lib/python3.4/site-packages/oled/Font.py", line 119, in __init__
    self.scale = scale
  File "/home/mauricio/virtual-envs/asistente/lib/python3.4/site-packages/oled/Font.py", line 127, in scale
    self.scale = new_scale
  .
  .
  .
  File "/home/mauricio/virtual-envs/asistente/lib/python3.4/site-packages/oled/Font.py", line 127, in scale
    self.scale = new_scale
  File "/home/mauricio/virtual-envs/asistente/lib/python3.4/site-packages/oled/Font.py", line 127, in scale
    self.scale = new_scale
RuntimeError: maximum recursion depth exceeded
```

**Comment by selfbg** [comment]
Try changing

    def __init__(self, scale=1):
        """
        Default constructor
        :param scale: multiplier for 5x7 font
        """
        self.scale = scale

To:

     def __init__(self, set_scale=1):
        """
        Default constructor
        :param scale: multiplier for 5x7 font
        """
        self.scale = set_scale
`

**Comment by gusantor** [comment]
Hello, thanks for your answer, but it seems doesn't work

I did both changes (line 113 & line 119) at

`/home/mauricio/virtual-envs/asistente/lib/python3.4/site-packages/oled/Font.py
`
same error as before

```
File "/home/mauricio/virtual-envs/asistente/lib/python3.4/site-packages/oled/Font.py", line 127, in scale
    self.scale = new_scale
RuntimeError: maximum recursion depth exceeded
```

thank you

**Comment by mcintst** [comment]
I have exactly the same issue :-(
Is there any solution?

**Comment by gusantor** [comment]
Hello, nope, at least not for me, sorry

**Comment by mcintst** [comment]
Solution: in Font.py change the following lines:

    @property
    def scale(self):
        return self._scale

    @scale.setter
    def scale(self, new_scale):
        self._scale = new_scale

(for more details about the issue see https://stackoverflow.com/questions/18989362/python-property-setting-from-list-causes-maximum-recursion-depth-exceeded )
-Stuart.

**Comment by selfbg** [comment]
Fixed in 56c426524f05b6ba5f2b770606e95ce5026d98e2
